code/modules/mob/living/carbon/xenomorph/mutators/strains/spitter/marker.dm

```diff
@@ -0,0 +1,39 @@
+/datum/xeno_mutator/marker
+	name = "STRAIN: Spitter - Marker"
+	description = "You weaken your corrosive acid for pheremones to support your sisters and acid to weaken your foes"
```

Comment:
This is an inaccurate description, and having 'pheromones' would give people the wrong idea. Should be polished up before merging.